ARitz-Cracker

```diff
@@ -0,0 +1,11 @@
+2018-10-21
```

Comment:
Hi @ARitz-Cracker — the PR looks good, thanks!

We also need your e-mail, the second part of signing the CLA before we can merge this: https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sending-the-email

Please let us know once that's done and we'll get you sorted :)

Thank you!